### ISSUE TITLE
ui language does not change no matter what i try

### ISSUE THREAD

**Issue by linuarM** [RVC-Boss/GPT-SoVITS#1834]
i downloaded the windows 10 link. then open the bat file and everything is chinese as followed in the picture too.

![saveee](https://github.com/user-attachments/assets/c815715e-a2f0-4701-89a4-7eaeaf28cfe1)

i thought it could be because my windows gui language being japanese. but later i turned it to english and restarted my computer it stayed the same.

i tried changing the GPT-SoVITS-beta\GPT-SoVITS-beta0706\tools\i18n\locale files by deleting them and keeping the one i need. but it did not work. 

i tried re downloading the GPT-SoVITS but it still is the same.
what should i do?

**Comment by XXXXRT666** [comment]
In the .bat file, it's `python webui.py <language>`

**Comment by XXXXRT666** [comment]
Maybe you can use en_US or ja_JP

**Comment by linuarM** [comment]
my .bat ui is 

runtime\python.exe webui.py ja_JP
pause

i tried en_US and tr_TR too but somehow the result is still chinese.
maybe im writing .bat file in a wrong way?

**Comment by linuarM** [comment]
> Maybe you can use en_US or ja_JP

my .bat ui is

runtime\python.exe webui.py ja_JP
pause

i tried en_US and tr_TR too but somehow the result is still chinese.
maybe im writing .bat file in a wrong way?

**Comment by XXXXRT666** [comment]
then delete the zh_CN.json

**Comment by XXXXRT666** [comment]
If it's still Chinese, maybe there is some cache

**Comment by XXXXRT666** [comment]
If you delete all the i18n files, it will remains Chinese

**Comment by linuarM** [comment]
> If you delete all the i18n files, it will remains Chinese

yes after i deleted the zh_CN.json it still remained the same. where can that caches could be?

**Comment by XXXXRT666** [comment]
Just unzip a new packages, and change the zh_CN to en_US in the BAT file

**Comment by linuarM** [comment]
> en_US

i unzipped and editted .bat added en_US but it didnt work..

**Comment by XXXXRT666** [comment]
Rename the file en_US to zh_CN then, and show the logs, or you could `print(i18n)` in the webUI to check the i18n language

**Comment by linuarM** [comment]
> Rename the file en_US to zh_CN then, and show the logs, or you could `print(i18n)` in the web.py to check the i18n language


it did not change when changed en_US to zh_CN. how can i show the logs? or where is the web.py file exactly?
